drivers/dma/dma_stm32.h

```diff
@@ -7,6 +7,30 @@
 #ifndef DMA_STM32_H_
 #define DMA_STM32_H_
 
+#if DT_NODE_HAS_STATUS(DT_DRV_INST(0), okay)
```

Comment:
headers should ideally be self-contained, so devicetree API should be included here

Reply:
Probably could be move to the .c file.